src/cascadia/TerminalApp/App.cpp

```diff
@@ -445,7 +445,12 @@ namespace winrt::TerminalApp::implementation
         co_await winrt::resume_background();
 
         const auto settingsPath = CascadiaSettings::GetSettingsPath();
-        ShellExecute(nullptr, L"open", settingsPath.c_str(), nullptr, nullptr, SW_SHOW);
+
+        HINSTANCE res = ShellExecute(nullptr, nullptr, settingsPath.c_str(), nullptr, nullptr, SW_SHOW);
+        if ((INT_PTR)res <= 32)

```

Comment:
In additional to Michael's comment, I'd prefer a `static_cast<intptr_t>()` versus a C cast, but I'm otherwise totally willing to approve this.

Reply:
Let me change this and push again.

Reply:
Well apparently only 

> (reinterpret_cast<intptr_t>(res)

works, which I think does the same

Reply:
Looking at this some more I think that reinterpret_cast just works since it does not check anything and the static cast fails, hence I will keep it as it is. Feel free to change it.